client/src/components/auth/SignUp.js

```diff
@@ -40,9 +40,27 @@ export default class SignUp extends Component {
     }
 
     try {
-      await this.props.login()
-    } catch (error) {
-      console.log(error.message)
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': "*" },
+        body: JSON.stringify({ username: this.state.name, email: this.state.email, password: this.state.password })
+      };
+
+      fetch('http://localhost:3002/users', requestOptions)
```

Comment:
```suggestion
      fetch('/users', requestOptions)
```
You don't need the host because we have the proxy option set up https://github.com/hatchways/team-fettuccini/blob/feature-auth-integration/client/package.json (line 22)

Same for